queries.sql

```diff
@@ -159,10 +164,19 @@ INSERT OR IGNORE INTO pkgs
 (category, dir)
 VALUES (?, ?);
 
+-- name: PutMaintainer :exec
+INSERT OR IGNORE INTO maintainers
+(pkg_maintainer)
+VALUES (?);
```

Comment:
Can you make this `RETURNING maintainer_id`, or does that not work with `INSERT OR IGNORE`?